l10n_it_fiscalcode/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,12 @@
+* Davide Corio <davide.corio@abstract.it>
+* Luca Subiaco <subluca@gmail.com>
+* Simone Orsi <simone.orsi@domsense.com>
+* Mario Riva <mario.riva@agilebg.com>
+* Mauro Soligo <mauro.soligo@katodo.com>
+* Giovanni Barzan <giovanni.barzan@gmail.com>
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Roberto Onnis <onnis.roberto@gmail.com>
+* Franco Tampieri <franco@tampieri.info>
+* Andrea Cometa <info@andreacometa.it>
+* Andrea Gallina <a.gallina@apuliasoftware.it>
+* Alex Comba <alex.comba@agilebg.com>
```

Comment:
You could add your name in the contributors list

Reply:
Done in https://github.com/OCA/l10n-italy/pull/607/commits/db7b6b1299ce7ca3d5e00c5cd342b70edf957446